### DISCUSSION TITLE
Expeditions 2 and 4- Atlas Issues

### DISCUSSION THREAD

**Discussion by oowatie** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#25] [General]
First of all, thank you to leodium and anyone else that has helped providing this content and making it work. I've really enjoyed catching up with all of it since I only started playing this year.

So far, everything has been working well for me except for anything Atlas-related (at least as far as I've gotten). Polo, when asked to provide Atlas coordinates, says that he is providing them, but they never show up, and the Atlas option never appears in the Galaxy map. For expedition 2, I just skipped that step and finished without completing everything (since I was able to get the Normandy anyway). But for expedition 4, the lack of an Atlas path borks all the steps after it, so I can't finish the expedition at all.

If anyone has experienced this, or has any suggestions to fix it, I would love to hear them. Thanks in advance.

**Comment by cwmonkey** [comment]
@oowatie Ask him for black hole coordinates and he will give you black hole AND Atlas coordinates. This is in the readme for 2, but not 4 @leodium
